index.js

```diff
@@ -51,7 +51,9 @@ module.exports = opts => {
 	opts = opts || {};
 
 	if (process.platform === 'win32') {
-		return [];
+		let cp = require("child_process");
+		let dk = cp.spawnSync("doskey", ["/history"]);
+		return dk.stdout.toString().replace(/\n/g, '').split('\r') || [];
```

Comment:
```suggestion
		return dk.stdout.trim().replace(/\n/g, '').split('\r') || [];
```

Reply:
Why are you removing `\n`? Does it output `\n\r` or `\r` if so, it's better to just split on either.

Reply:
Noted and changed.